sources/packages/api/src/route-controllers/restriction/restriction.aest.controller.ts

```diff
@@ -104,12 +107,10 @@ export class RestrictionController extends BaseController {
    * @param restrictionCategory Selected Category from category list.
    * @returns Reasons option list.
    */
-  @Groups(UserGroups.AESTUser)
-  @AllowAuthorizedParty(AuthorizedParties.aest)
   @Get("/reasons/options-list/category/:restrictionCategory")
```

Comment:
suggestion `@Get("category/:restrictionCategory/reasons")` and I think function name can be `getCategoryReasons`